src/main/docker/scripts/start.sh

```diff
@@ -15,5 +15,5 @@
 # limitations under the License.
 #
 
-export JAVA_HOME=/usr/share/elasticsearch/jdk
+export JAVA_HOME=$JRE_HOME
```

Comment:
Prior to this we probably did not need JAVA_HOME here, since it defaulted to the bundled install.  Now we do.